### PR TITLE
New Lightroom CheatSheet

### PR DESCRIPTION
**Pull request by biscuitNinja**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Addition of a Cheat Sheet for Adobe Lightroom

###### Which issues (if any) does this fix?

N/A

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/lightroom_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @biscuitNinja

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/lightroom.json [hunk 1]
```diff
@@ -0,0 +1,996 @@
...
+                "key": "[Ctrl] [Alt] [6]",
+                "val": "Open Print module"
+            }, {
+                "key": "[Ctrl] [Alt] [7]",
+                "val": "Open Web module"
+            }, {
+                "key": "[Ctrl] [Alt] [↑]",
+                "val": "Toggle between previous/current module"
+            }, {
+                "key": "[Ctrl] [Alt] [↔]",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/lightroom.json]: Is there a mistake with arrow here?

File: share/goodie/cheat_sheets/json/lightroom.json [hunk 2]
```diff
@@ -0,0 +1,996 @@
...
+                "key": "[Ctrl] [Left-click] panel section heading",
+                "val": "Contract/Expand all sections in panel"
+            }, {
+                "key": "[Alt] [Left-click] panel section heading",
+                "val": "Expand current section and contract all other sections in panel (solo mode)"
+            }, {
+                "key": "[Shift] [Left-click] panel section heading",
+                "val": "App panel section to solo mode"
+            }, {
+                "key": "tab",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/lightroom.json]: Tab will be good

**Comment by biscuitNinja** [comment]
@jophab I'll get the s/tab/Tab/ and the arrow fixed tonight probably, if not soon after

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=lightroom cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/lightroom_cheat_sheet_mobile.png?IXM1VyaRKO)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=lightroom cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/lightroom_cheat_sheet_mobile.png?3VXjcEgx0M)

**Comment by biscuitNinja** [comment]
@jophab Changes you identified now complete

**Comment by zekiel** [comment]
woop woop! looks great!

**Comment by daxtheduck** [comment]
@biscuitNinja this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**